docs/get-started.md

```diff
@@ -142,7 +142,41 @@ output of `docker-machine ls`.
 
 If you are finished using a host for the time being, you can stop it with
 `docker-machine stop` and later start it again with `docker-machine start`.
-Make sure to specify the machine name as an argument:
 
-    $ docker-machine stop dev
-    $ docker-machine start dev
+    $ docker-machine stop default
+    $ docker-machine start default
+
+## Operating on machines without specifying the name
+
+Some commands will assume that the specified operation should be run on a
+machine named `default` (if it exists) if no machine name is passed to them as
+an argument.  Because using a local VM named `default` is such a common pattern,
+this allows you to save some typing on Machine commands that may be frequently
+invoked.
+
+For instance:
+
+    $ docker-machine stop
+    Stopping "default"....
+    Machine "default" was stopped.
+
+    $ docker-machine start
+    Starting "default"...
+    (default) Waiting for an IP...
+    Machine "default" was started.
+    Started machines may have new IP addresses.  You may need to re-run the `docker-machine env` command.
```

Comment:
This implies that `docker-machine env` will work, but then the list below doesn't list it.

if `docker-machine env` doesn't work with out the machine name, then it needs to say so here too, else its confusing.